src/components/users/login/LoginPage.js

```diff
@@ -9,18 +9,9 @@ import { loggedInUserContext } from '../../../Helper/loggedInUserContext';
 const LoginPage = () => {
   const { setLoggedInUser } = useContext(loggedInUserContext);
   const [user, setUser] = useState(userBlankData());
-  const [loginResponse, setLoginResponse] = useState({
-    data: { token: '', user: {} },
-  });
   let navigate = useNavigate();
   const [loginError, setLoginError] = useState(false);
-
-  useEffect(() => {
```

Comment:
What is the reasoning for removing this useEffect?

I believe before it was checking to see if the user was logged in?